usaspending_api/search/v2/views/search.py

```diff
@@ -436,7 +436,7 @@ def post(self, request):
         self.scope = json_request["scope"]
         self.filters = json_request.get("filters", None)
         self.geo_layer = json_request["geo_layer"]
-        self.geo_layer_filters = json_request["geo_layer_filters"]
+        self.geo_layer_filters = json_request.get("geo_layer_filters", None)
```

Comment:
I tried to make it match the `filters` param as closely as possible. Should we change the `filters` to do the same thing in the `AWARD_FILTER` object?

Reply:
Retrieving the filters is a work-in-progress since we do want to enforce a key `filters` with a python dict as the value, but don't want to list all of the sub-dict rules within that model. There is an experimental TinyShield datatype of "Schema" which will validate that the key is present but doesn't look at the value for that key.

SpendingOverTime's `group` model might be a better example of how `geo_layer_filters` should behave for the meantime.

Reply:
Disregard previous comments, the behavior I discussed has not been implemented yet and is out-of-scope of this PR